core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -241,7 +245,12 @@ public RemoteLogManager(RemoteLogManagerConfig rlmConfig,
 
         indexCache = new RemoteIndexCache(rlmConfig.remoteLogIndexFileCacheTotalSizeBytes(), remoteLogStorageManager, logDir);
         delayInMs = rlmConfig.remoteLogManagerTaskIntervalMs();
-        rlmScheduledThreadPool = new RLMScheduledThreadPool(rlmConfig.remoteLogManagerThreadPoolSize());
+        rlmScheduledThreadPool = new RLMScheduledThreadPool(rlmConfig.remoteLogManagerThreadPoolSize(),
```

Comment:
Could we rename the variable to something meaningful to distinguish with other 2 threadPools? Maybe `rlmCopyThreadPool`?

Reply:
I wanted to rename it. Is it okay to rename the threadpool name that will show up in the logs? Would that break any existing contract?

Reply:
No, it won't break the contract. No worries.